game/debuffHeal.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DEBUFF_HEAL_H
+#define DEBUFF_HEAL_H
+
+#include <string>
+#include <vector>
+
+#include "item.h"
+#include "debuff.h"
+
+class Player;
+
+class Potion: public Item {
```

Comment:
how come this class is named `Potion` when the file name is `debuffHeal`? Our UML shows both classes